src/components/internal-header/template.njk

```diff
@@ -2,20 +2,45 @@
-                                <path d="M67.47,7.72a59.76,59.76,0,1,0,59.75,59.75A59.75,59.75,0,0,0,67.47,7.72m67.47,59.75A67.47,67.47,0,1,1,67.47,0a67.47,67.47,0,0,1,67.47,67.47" />
-                        </svg>
-                        <span class="hmrc-internal-header__logo-text">{% if params.language == 'cy' %}
+					{% if params.useTudorCrown | default(true) %}
+            <!--[if gt IE 8]><!-->
```

Comment:
Again, spotted that the IE8 stuff has crept back in.

Reply:
done!